conda/notices/types.py

```diff
@@ -28,6 +28,17 @@ class ChannelNotice(NamedTuple):
     expired_at: datetime | None
     interval: int | None
 
+    def notice_to_json_notice(self):
```

Comment:
A standard name for such a method is `to_dict` or `for_json`. We have many classes that implement `to_dict`, so I would suggest to use that.
```suggestion
    def to_dict(self):
```